src/hybrid_a_star_planner/path_planner/hybrid_a_star.py

```
@@ -59,7 +61,26 @@ def get_h_cost(x_idx: int, y_idx: int) -> float:
+        # ==== 动态障碍物惩罚 ====
+        # 预测到达该点的时间（可用g_cost近似）
+        t = g_cost
+        penalty = 0.0
+        for obs in getattr(map_params, "dynamic_obstacles", []):
+            obs_x, obs_y = obs.get_position(t)
+            #dist = np.hypot(x_idx - obs_x, y_idx - obs_y)
+            # 将栅格索引转换为世界坐标          
+            world_x = x_idx * map_params.xy_resolution
+            world_y = y_idx * map_params.xy_resolution
+            dist = np.hypot(world_x - obs_x, world_y - obs_y)
+            if dist < 0.5:  # 0.5为安全距离阈值，可调整
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)